tests/integration/targets/python_requirements_info/tasks/main.yml

```diff
@@ -19,6 +19,21 @@
       - "'python_version' in basic_info"
       - basic_info.python_version_info == ansible_python.version
 
+# See https://pip.pypa.io/en/stable/reference/requirement-specifiers/
+- name: ensure module supports PEP-508 specification
+  python_requirements_info:
+    dependencies:
+      - SomeProject
+      - SomeProject == 1.3
+      - SomeProject >= 1.2, < 2.0
+      - SomeProject[foo, bar]
+  register: dep_info
+
+- name: ensure python_requirements_info returns desired info
+  assert:
+    that:
+    - dep_info is not failed
```

Comment:
IMHO this makes more sense in the 731-parse-project-extra-syntax.yml file. And shouldn't we use an existing package instead of `SomeProject` ?

Reply:
Does it ? Because it should be handled by the general module normally :)

Regarding the SomeProject, given that we assess only the validity of the requirement format, I'm not sure which added-value we could benefit there. WDYT ?

Reply:
> Does it ? Because it should be handled by the general module normally :)

That came from the fact that we are testing some extra deps in there, but I see what you mean. Yeah, I am OK with it as-is.

> Regarding the SomeProject, given that we assess only the validity of the requirement format, I'm not sure which added-value we could benefit there. WDYT ?

Ok,  fair enough. But me lazily not going through the whole thing again, are testing elsewhere that all the extras specified are present, when responding?